src/pytest_mock/plugin.py

```diff
@@ -102,7 +110,8 @@ def stopall(self) -> None:
         times.
         """
         for p, m in reversed(self._patches_and_mocks):
-            p.stop()
+            if p:
```

Comment:
Let's be more explicit here:

```suggestion
            if p is not None:
```

Given `p` is Any, is theoretically possible for it to implement `__bool__` and return `False` for some reason.